components/Common/LanguageDropDown/index.module.css

```diff
@@ -0,0 +1,39 @@
+.iconWrapper {
+  @apply h-9
+	w-9
```

Comment:
Hmm, the indentation is wrong here again 👀  (on all these lines)

Reply:
Hey, the indentation is correct in my local branch which I pushed.

![image](https://github.com/nodejs/nodejs.org/assets/66557682/217c0530-bb4e-4a0d-8c0c-a2f2794d373a)

I am confused about what's happening.

Reply:
Hmm, just viewed the prettierrc file and saw that the indentation with spaces is preferred but the css file uses tabs instead. with tab size of 2.
Viewing the file on github, it uses tab size of 2
![image](https://github.com/nodejs/nodejs.org/assets/66557682/168a95c9-6d0b-4ac9-8c18-bf3180435460)

But locally in my setup, its 4.

Reply:
> I have the feeling you used tabs here instead of spacing. Try to use just two spaces.
Sure